maldoca/base/get_runfiles_dir.cc

```diff
@@ -56,7 +56,10 @@ std::string GetExecutablePath() {
   HMODULE hModule = GetModuleHandleW(NULL);
   WCHAR wc_file_path[MAX_PATH] = {0};
   GetModuleFileNameW(hModule, wc_file_path, MAX_PATH);
-  std::string file_path = base::WideToUTF8(wc_file_path);
+  // see
+  // https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd
```

Comment:
fine for me, however seems this solution also has some issues: https://docs.bazel.build/versions/main/skylark/windows_tips.html#paths

they recommend using "--output_user_root=D:\bzl" or so.

if u wanna us the long paths, plz also add it to the project readme's requirements section :)

Reply:
This is just for testing so don't really care. This is more portable;  presumably chrome may have the same long path problem and they don't use bazel.

Reply:
k, added a todo for me to mention it in the reamdme.